exercises/luhn/example.cpp

```diff
@@ -0,0 +1,32 @@
+#include "luhn.h"
+
+using namespace std;
+
+namespace luhn {
+bool valid(string const& input_str) {
```

Comment:
We should have a standard way of writing this... I don't mind if it's `const string` or `string const`, but there should be a single way through the whole repo.

Reply:
I believe we've been pushing everything to use `string const&` for reference inputs, though `const int` is used for local variables, so this should be fine.

Yes it's somewhat inconsistent, but this style is consistent with the rest of the repo.